src/sagemaker/workflow/pipeline.py

```diff
@@ -824,15 +828,34 @@ def list_steps(self):
         """Describes a pipeline execution's steps.
 
         Returns:
-             Information about the steps of the pipeline execution. See
+             Information about the steps of the pipeline execution in a Pandas table. See
              `boto3 client list_pipeline_execution_steps
              <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/\
 sagemaker.html#SageMaker.Client.list_pipeline_execution_steps>`_.
         """
+        try:
+            import pandas as pd
+        except ImportError:
+            raise ImportError(
+                """
+                Please install the pandas library to use this command.
+                Run 'pip install pandas' and reinvoke command.
+                """
+            )
+
         response = self.sagemaker_session.sagemaker_client.list_pipeline_execution_steps(
             PipelineExecutionArn=self.arn
         )
-        return response["PipelineExecutionSteps"]
+
+        pd.set_option("display.max_colwidth", None)
```

Comment:
We will print tables for list pipelines and list executions right?
We'd better extract this part (850 - 858) to a separate method to reuse it for all list methods.

Let's add the improvement for list pipelines and list executions in this PR as well